src/include/simeng/models/outoforder/Core.hh

```diff
@@ -44,12 +47,16 @@ class Core : public simeng::Core {
   const ArchitecturalRegisterFileSet& getArchitecturalRegisterFileSet()
       const override;
 
+  /** Send a syscall to the system syscall handler. */
+  void sendSyscall(OS::SyscallInfo syscallInfo) const override;
+
+  /** Communicate the result of a syscall to the core's active exception
```

Comment:
The comment in `simeng/Core.hh` reads: "This method receives the result of an initiated syscall and communicates the result to the exception handler for post-processing", however the current comment still says "active" syscall handler.

Reply:
Outdated